lib/mix/test/mix/dep_test.exs

```diff
@@ -197,49 +205,56 @@ defmodule Mix.DepTest do
              deps: [{:git_repo, "0.2.0", git: MixTest.Case.fixture_path("git_repo"), optional: true}]]
           end
         end
-        """
+        """)
 
-        assert Enum.map(Mix.Dep.loaded([]), &(&1.app)) == [:git_repo, :deps_repo]
+        assert Enum.map(Mix.Dep.loaded([]), & &1.app) == [:git_repo, :deps_repo]
       end
-    end
+    end)
   end
 
   test "nested deps with optional dependencies and cousin conflict" do
-    with_deps [{:deps_repo1, "0.1.0", path: "custom/deps_repo1"},
-               {:deps_repo2, "0.1.0", path: "custom/deps_repo2"}], fn ->
-      in_fixture "deps_status", fn ->
-        File.mkdir_p!("custom/deps_repo1")
-        File.write! "custom/deps_repo1/mix.exs", """
-        defmodule DepsRepo1 do
-          use Mix.Project
+    with_deps(
+      [
+        {:deps_repo1, "0.1.0", path: "custom/deps_repo1"},
+        {:deps_repo2, "0.1.0", path: "custom/deps_repo2"}
+      ],
+      fn ->
+        in_fixture "deps_status", fn ->
```

Comment:
Please break out these nested anonymous functions into named functions. Also break out the list above to a variable.